### PR TITLE
Enable errors when setting up a course

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
None.

To test this, create new instructors account. Create an assignment, when setting up the course try to enter a course code that you have used in other accounts. That should show an error. 
Once you created few courses, try to delete them. Course that has assignments associated with it should show error when the user tries to delete it.

### CODE REVIEW

File: instructor/common.py [hunk 1]
```diff
@@ -210,9 +210,6 @@ def course_setup(request):
     if course_selected:
         return redirect("common_course_modify")
 
-    formset = CourseFormSet(
```
Comment (noisecapella) [inline instructor/common.py:213]: Will `formset` ever be undefined when it's referenced below?

**Review by noisecapella — CHANGES_REQUESTED** [review]
I'm getting this error when clicking 'Save and continue' on the assignment setup:

screenshot from 2016-12-09 11-48-42

File: instructor/common.py (hunk 1 above)
Comment (annagav) [reply instructor/common.py:213]: I think I fixed it.

**Review by noisecapella — APPROVED** [review]
Looks great :+1:

**Pull request merged**